AlembicImporter/Packages/com.unity.formats.alembic/Editor/Misc/BoolDrawer.cs

```diff
@@ -1,7 +1,8 @@
 using UnityEngine;
+using UnityEngine.Formats.Alembic.Sdk;
 using UnityEditor;
 
-namespace UTJ.Alembic
+namespace UnityEditor.Formats.Alembic.Importer
 {
```

Comment:
This could also be in a "Util" namespace but since only the importer uses it I left it there